### DISCUSSION TITLE
Cannot use DoH in «managed» browsers (repro of #832)

### DISCUSSION THREAD

**Discussion by dksarangoung** [bromite/bromite#2264] [General]
I am using an Amazon tablet with Fire OS 7.3.2.4. DoH used to work with Bromite in spite of running in a «managed» environment. However, I can no longer use DoH on 104.0.5112.63. This is a regression of #832 which was fixed last year IIRC. The issue should be reopened and fixed.

**Comment by csagan5** [comment]
You can open a new issue; discussions are not for reporting issues.
